### ISSUE TITLE
ui5-illustrated-message: not accessible

### ISSUE THREAD

**Issue by ee92** [SAP/ui5-webcomponents#4878]
### **Bug Description**
The ui5-illustrated-message component is not announced by screen readers and is not accessible by tabbing

### **Expected Behavior**
It should announce the title and subtitle text when it renders, or at least be accessible by tabbing to it

### **Steps to Reproduce**
1. Have a screen reader like JAWS or NVDA turned on
2. Go to https://sap.github.io/ui5-webcomponents/playground/components/IllustratedMessage/
3. Notice nothing is announced by the screen reader
4. Notice it is not accessible by tabbing

### **Context**
- UI5 Web Components version: 1.2.1
- Affected component: ui5-illustrated-message

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

### **Stakeholder Info** (if applicable)

- Organization: SuccessFactors
- Business impact: Error message not accessible to screen reader users

Thank you :)

**Comment by PetyaMarkovaBogdanova** [comment]
Hi colleagues,
can you, please, check if we should make the control more accessible by screen raeders?

Kind regrads,
Petya Markova.
(UI5 Dispatcher)

**Comment by nnaydenow** [comment]
Hi @ee92,

ui5-illustrated-message is non-interactive components and it should not provide accessibility by default. Its accessibility should be handled by its parent. 

For example if we have ui5-illustrated-message inside table and it should illustrated that table should not have data. Since the no data text should be handled by table it means that the accessibility of ui5-illustrated-message should be handle by table too.

Regards,
Nayden
